aspnetcore/fundamentals/servers/kestrel/includes/http-8.md

```diff
@@ -0,0 +1,69 @@
+:::moniker range="= aspnetcore-8.0"
```

Comment:
This file is the V8 version, a copy of aspnetcore/fundamentals/servers/kestrel/http3.md